book/MovingToTheGPU.html

```diff
@@ -3897,4 +3897,5 @@
 <link rel='stylesheet' href='../style/book.css'>
 <style class="fallback">body{visibility:hidden;white-space:pre;font-family:monospace}</style>
 <script src="markdeep.min.js"></script>
+<script crossorigin="anonymous" src="https://morgan3d.github.io/markdeep/latest/markdeep.min.js"></script>
```

Comment:
I've added script tags to the latest markdeep library. 
It should be noted that without the `crossorigin="anonymous"` attribute the script would not load for anyone
due to [CORS](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/crossorigin).
This is also an issue of the markdeep starter templates, which should be addressed.

Reply:
Remove the local script line and the crossorigin attribute, and you'll find that the script loads just fine.

Reply:
Haha, I'm guessing you don't start a local dev server when you test this?
When you open an HTML file using the file:/// protocol, CORS restrictions are typically not enforced by browsers.
Refer: https://developer.mozilla.org/en-US/docs/Web/Security/Same-origin_policy#file_origins

Reply:
See https://hollasch.net/rtw/books/RayTracingInOneWeekend.html. This version has no local script present, nor referenced, pulling only from the latest Markdeep version. Page loads and renders without issue (Brave browser).

Reply:
More information on the `crossorigin` attribute: https://stackoverflow.com/questions/41069330/with-script-crossorigin-anonymous-why-is-a-script-blocked-by-cors-policy

It looks like the only reason to add `crossorigin='anonymous'` is "if we care about getting error information for the script being loaded", which we don't. Default behavior is specified by explicitly omitting the `crossorigin` attribute.

Reply:
You're right, but for some reason I get `NS_ERROR_FAILURE` on firefox when I first load the html page. it opens after i load it again. not sure what's the reason

Reply:
That definitely sounds annoying. We treat the HTML docs as standalone documents, so don't normally fire up a web host just to read them. This is how we expect our readers to view the local documents as well.